### PR TITLE
fix hover + tooltips for search v3

### PR DESCRIPTION
**Pull request by zanderz**
@keybase/react-hackers

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 3ef1f360ba81...7b16418741eb introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1498612001701-3ef1f360ba81-7b16418741eb-linux/Tracker-New_user_Scroll_1-3ef1f360ba81.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1498612001701-3ef1f360ba81-7b16418741eb-linux/Tracker-New_user_Scroll_1-7b16418741eb.png)  
   Tracker-New_user_Scroll_1 comparison
</summary>

File: shared/searchv3/services-filter/index.js [hunk 1]
```diff
@@ -109,7 +118,7 @@ class Filter extends Component<void, Props, {hoveredService: ?Constants.Service}
             service={service}
             selected={service === this.props.selectedService}
             hovering={service === this.state.hoveredService}
-            onHover={this._hoverChanged}
+            onHover={isHovering => this._hoverChanged(service, isHovering)}
```
Comment (chrisnojima) [inline shared/searchv3/services-filter/index.js]: why did you remove the service and make this closure? we want to avoid having closures as props like this

File: shared/searchv3/services-filter/index.js [hunk 2]
```diff
@@ -118,4 +127,20 @@ class Filter extends Component<void, Props, {hoveredService: ?Constants.Service}
   }
 }
 
+const serviceTooltipStyle = {
+  ...transition('opacity'),
+  backgroundColor: globalColors.black_40,
+  borderBottom: '2px solid white',
```
Comment (chrisnojima) [inline shared/searchv3/services-filter/index.js]: this likely doesn't work in react native

File: shared/searchv3/services-filter/index.js [hunk 3]
```diff
@@ -118,4 +127,20 @@ class Filter extends Component<void, Props, {hoveredService: ?Constants.Service}
...
+  borderRadius: 65,
+  color: globalColors.white,
```
Comment (chrisnojima) [inline shared/searchv3/services-filter/index.js]: this isn't centered correctly

image

File: shared/searchv3/services-filter/index.js (hunk 2 above)
Comment (zanderz) [inline shared/searchv3/services-filter/index.js]: which part exactly? what should I do?

Comment (chrisnojima) [inline shared/searchv3/services-filter/index.js]: run this in RN and you'll see. RN has sep. values for this, not the short hand ones so you have to do
borderBottomColor: globalColors.white,
borderBottomWidth: 2, etc

**Comment by zanderz** [comment]
Mobile works now but the tooltip is ugly/in the wrong place so I disabled it for the time being - is there even really supposed to be one on Mobile?

**Comment by chrisnojima** [comment]
There isn't I assume

File: shared/common-adapters/clickable-box.js.flow [hunk 4]
```diff
@@ -14,6 +14,11 @@ export type Props = {|
   feedback?: boolean,
   // mobile only
   activeOpacity?: number,
+  // desktop only
```
Comment (chrisnojima) [inline shared/common-adapters/clickable-box.js.flow]: are these used anymore?

Comment (zanderz) [inline shared/common-adapters/clickable-box.js.flow]: Flow complains about the shared/common-adapters/clickable-box.desktop.js changes without this

**Comment by zanderz** [comment]
Rebased

File: shared/common-adapters/clickable-box.js.flow (hunk 4 above)
Comment (chrisnojima) [inline shared/common-adapters/clickable-box.js.flow]: yeah but, where is that used in this pr?

Comment (zanderz) [inline shared/common-adapters/clickable-box.js.flow]: Those calls need to be forwarded like this, otherwise it doesn't work, I just tried it again. Marco found this when I pestered him for some help.

Comment (chrisnojima) [inline shared/common-adapters/clickable-box.js.flow]: ok so the plumbing was already there, k tnx
